components/unstoppable_domains/test/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/config.gni")
+import("//brave/components/unstoppable_domains/buildflags/buildflags.gni")
+import("//testing/test.gni")
+
+source_set("unit_tests") {
+  testonly = true
+  if (unstoppable_domains_enabled) {
+    sources = [
+      "//brave/components/unstoppable_domains/utils_unittest.cc",
+      "//net/dns/dns_transaction_unittest.cc",
+    ]
+    deps = [
+      "//base/test:test_support",
+      "//brave/components/unstoppable_domains",
+      "//chrome/test:test_support",
```

Comment:
looks illegal, it seems you'll have to move the test out of /components

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/cf30603be56f77af3d8bd829007e66246640293f